src/cws_pattern.js

```diff
@@ -101,14 +110,21 @@ function get_crx_url(extensionID_or_url) {
     if (match) {
         return 'https://' + match[1] + '/firefox/downloads/file/' + match[2] + (match[3] || '/addon.xpi');
     }
-    // Chrome Web Store
+
     match = get_extensionID(extensionID_or_url);
     var extensionID = match ? match : extensionID_or_url;
 
     if (!/^[a-z]{32}$/.test(extensionID)) {
         return extensionID_or_url;
     }
+    
+    // Edge Addons Store
+    var is_mea_extension = (mea_pattern.exec(extensionID_or_url) || mea_download_pattern.exec(extensionID_or_url));
```

Comment:
Change to:

```
match = mea_pattern.exec(url) || mea_download_pattern.exec(url);
```

and use `match[1]` below as needed.

and move the whole new block before the original location of `// Chrome Web Store`.

Reply:
Done